src/ui/components/DashKit/plugins/GroupControl/Control/Control.tsx

```diff
@@ -98,10 +99,10 @@ export const Control = ({
     getDistincts,
     onChange,
     needReload,
-    cancelSource,
     workbookId,
 }: ControlProps) => {
     const [prevNeedReload, setPrevNeedReload] = React.useState(needReload);
+    const [requestCancellation, setRequestCancellation] = React.useState<CancelTokenSource>();
```

Comment:
Do we really need it as state? I don't see any needs for component to redraw while this object updates, why not use ref?